spec/database_handler_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+require_relative '../lib/database_handler'
+
+describe Database do
+  describe '.connect' do 
+    it 'connection to database' do 
+      expect(Database.connect).to be_a_instance_of(PG::Connection)
+    end 
+  end
+
+  describe '.all' do
+    it 'returns the whole database' do 
+      Database.add(message: "hello")
+      expect(Database.all.first['messages']).to eq 'hello'
+    end   
```

Comment:
It might be worth testing that `Database.all` also works when there is more than one message in the db